index.html

```diff
@@ -3440,20 +3440,20 @@ <h4 id="usage_of_ruby">
                   <p its-locale-filter-list="en" lang="en"><a href="#term.jukugo-ruby" class="termref">Jukugo-ruby</a>. Pronunciation is indicated for each <a class="characterClass" href="#cl-19">ideographic character (cl-19)</a>, but the positioning takes into account the fact that together they make up a <a href="#term.compound-word" class="termref2nd">compound word</a> (see [[[#fig2_3_4]]]). The intention when using jukugo-ruby is to handle the <a class="characterClass" href="#cl-19">ideographic character (cl-19)</a> phrase as one object.</p>
                   <p its-locale-filter-list="ja" lang="ja">個々の<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>の読み方を示すと同時に，熟語単位での配置も考慮して配置する方法（[[[#fig2_3_4]]]）．この方法は<span class="index" id="d5e8948"><a href="#term.jukugo-ruby" class="termref">熟語ルビ</a></span>とよばれている．熟語ルビは，熟語を構成する<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>の1文字1文字の読み方を示すとともに，熟語としてのまとまりを持っているので，ルビもまとまりとして読める必要がある．熟語ルビは，熟語としてのまとまりを重視した配置方法といえる．</p>
                   <figure id="fig2_3_4"> 
-      <img its-locale-filter-list="en" src="images/img2_3_4.png" alt="Example of jukugo-ruby method. Ruby letters are attached to groups of ideographic characters in compound words." width="242" height="360">
+      <img its-locale-filter-list="en" src="images/img2_3_4.png" alt="Example of jukugo-ruby method. ruby characters are attached to groups of ideographic characters in compound words." width="242" height="360">
       <img its-locale-filter-list="ja" lang="ja" src="images_ja/img2_3_4.png" alt="熟語に付くルビを熟語ルビで処理した例" width="250" height="360">
                     <figcaption>
-        <span its-locale-filter-list="en" lang="en">Example of jukugo-ruby method. Ruby letters are attached to groups of ideographic characters in compound words.</span>
+        <span its-locale-filter-list="en" lang="en">Example of jukugo-ruby method. ruby characters are attached to groups of ideographic characters in compound words.</span>
```

Comment:
capitalize the first character?

Reply:
I'd say, yes, capitalise the r in ruby in all cases you noticed.